packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -794,7 +794,11 @@ async function handler(
   const seed = `${organizerUser.username}:${dayjs(reqBody.start).utc().format()}:${new Date().getTime()}`;
   const uid = translator.fromUUID(uuidv5(seed, uuidv5.URL));
 
-  const bookingLocation = getLocationValueForDB(locationBodyString, eventType.locations);
+  let bookingLocation = getLocationValueForDB(locationBodyString, eventType.locations);
+
+  if (dynamicUserList.length > 1 && defaultLocationUrl) {
+    bookingLocation = defaultLocationUrl;
+  }
```

Comment:
So if there dynamic users -> re set the bookingLocation to the defaultLocationUrl which is fetched above